_topics/en/v2-p2p-transport.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Version 2 P2P Transport
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - P2P Network Protocol
+  - Privacy Enhancements
+
+aliases:
+  - BIP151
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Version 2 (v2) P2P Transport** is a proposal to allow Bitcoin nodes
+  to communicate with each other over encrypted connections.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Some other changes to the communication protocol are also suggested,
+  such as allowing frequently-used protocol commands to be aliased to
+  shorted byte sequences to reduce bandwidth.
+
+  This proposal replaces the earlier [BIP151][] proposal.
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: v2 Transport Protocol proposed BIP
+      link: https://gist.github.com/jonasschnelli/c530ea8421b8d0e80c51486325587c52
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
```

Comment:
Perhaps add /en/newsletters/2019/06/12/#v2-p2p (Schnelli's presentation of v2 p2p at the Core dev meetup)